meteor/lib/lib.ts

```diff
@@ -946,6 +946,9 @@ export function unprotectString(protectedStr: ProtectedString<any> | undefined |
 export function unprotectStringArray(protectedStrs: Array<ProtectedString<any>>): string[] {
 	return (protectedStrs as any) as string[]
 }
+export function unDeepString<T extends ProtectedString<any>>(str: ReadonlyDeep<T> | PartialDeep<T>): T {
```

Comment:
should be unprotectStringDeep?

Reply:
should it? its converting `ReadonlyDeep<RundownId>` to `RundownId`, so its still a protected string

Reply:
`unDeepProtectedString`?